### PR TITLE
Show warning when invested amount is not maximum available amount

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

If the invested amount is not equal to maximum available amount we show yellow warning text below the input field

Screenshot from 2022-01-25 12-20-52

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @nenadV91 , great changes!
A tiny nitpick: it shows 100% of the investment amount, but actually it is not 100%. Could we show 99,99% instead? 
image

Video: https://watch.screencastify.com/v/26mQjbcnGBVjHkz3raJt

Also, as I mentioned in https://github.com/gnosis/cowswap/pull/2279#issuecomment-1021104827 , it would be nice to change warning's color for the light mode, I think.

File: src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx
Comment (W3stside) [inline src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx:28]: why change the name here, they are more declarative as `ErrorMessage` no?

**Review by W3stside — APPROVED** [review]

**Comment by alfetopito** [comment]
> Hey @nenadV91 , great changes! A tiny nitpick: it shows 100% of the investment amount, but actually it is not 100%. Could we show 99,99% instead? 

I don't think such a tiny amount (less than 0.01%) is relevant. I vote for leaving as is.

> Also, as I mentioned in [#2279 (comment)](https://github.com/gnosis/cowswap/pull/2279#issuecomment-1021104827) , it would be nice to change warning's color for the light mode, I think.

That will come, @biocom will get there 🤞

**Review by alfetopito — COMMENTED** [review]
Works great but once concern: we should either move this warning up (so it triggers first) or make a list of warnings

I say that because the native fee warning will hide this one:
Screen Shot 2022-01-25 at 09 17 37

**Review by anxolin — COMMENTED** [review]
Nice! all good other than what Leandro mentioned.

> make a list of warnings

I would go for that one. For the errors, with one we have enough I believe. But Warnings we should see all the warnings at the same time. 


Maybe @annamsgeorge or @avsavsavs want to touch that message. It's used for warning the users when they only make use of part of their claiming opportunity.

**Review by alfetopito — APPROVED** [review]

**Comment by nenadV91** [comment]
@anxolin Updated to an array of warnings

**Comment by biocom** [comment]
@nenadV91 Pushed a fix for the style.ts merge conflict, in case it helps.

**Comment by elena-zh** [comment]
Hey, now I see not readable warning messages in the dark mode: 
image

**Comment by nenadV91** [comment]
@elena-zh Good catch, my mistake, fixed! Maybe later on @michelbio can also take a look at this.

**Comment by elena-zh** [comment]
@nenadV91 , thanks!
Now it is readable. But let wait for @biocom for the final decision on the text colors.

**Review by elena-zh — APPROVED** [review]

**Comment by biocom** [comment]
Can review post merge to address it then.

File: src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx
Comment (anxolin) [inline src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx:403]: Is not this the same as 

`{ inputWarnings.length && (... ) }`

**Review by anxolin — COMMENTED** [review]
Cool

**Pull request merged**
